lib/common.bash

```diff
@@ -532,3 +532,354 @@ check_dockerfiles_images()
 		build_dockerfile_image "$image" "$dockerfile_path"
 	fi
 }
+
+###############################################################################
+
+# Misc functions.
+
+TESTS_REPO_DIR=$(realpath $(dirname "${BASH_SOURCE[0]}")/..)
+
+# sudo shortcut function with environment
+esudo() {
+  sudo -E PATH=$PATH "${@}"
+}
+
+# Get version of component from tests versions.yaml
+tests_get_version() {
```

Comment:
This is a duplication of `get_test_version()` in `.ci/lib.sh`.

There are tests like `integration/kubernetes/k8s-replication.bats` that load `load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"` to use `get_test_version()`. I suggest to do the same with SEV-* tests.